tasks/podman-setup.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Configure loginctl to prevent logout causing podman to exit
+  ansible.builtin.shell: loginctl enable-linger
+
+- name: Set podman variable
+  set_fact:
+    podman_conf: podman.conf
+
+- name: "Install Podman on RHEL8 server"
+  package:
+    name:
+      - "@container-tools" # install module 'container-tools'
+      - podman-docker
+    state: present
+
+- name: "set subuids and subguids to {{ user | default('cloud-user') }} user"
+  lineinfile:
+    path: "/etc/{{ item }}"
+    line: "{{ user | default('cloud-user') }}:10000:65536" # to solve podman error: "setup user: invalid argument" in rootless mode
+  loop:
+    - subuid
+    - subgid
+
+- name: "Copy podman configuration"
+  copy:
+    dest: "/etc/sysctl.d/{{ podman_conf }}"
+    src: "{{ podman_conf }}"
+
+- name: "Start podman service"
+  ansible.builtin.systemd:
+    state: started
+    name: podman.service
```

Comment:
1. Podman backend is enabled and started by `podman.socket`.
2. This starts the rootful podman while the PR says we're enabling rootless podman, we're currently not sure about whether we need rootful podman or rootless podman, but even with rootful podman, this is not sufficient, we'll need to grant the permission to unprivileged users.
3. To start rootless podman with a different user (which runs the playbooks), you login the user via `machinectl`.

Reply:
This script is still not ready to start rootless podman, it's pending on the decision of desgin part